.github/workflows/e2e_tests.yml

```diff
@@ -37,6 +37,6 @@ jobs:
           service_account_key: ${{ secrets.GCP_CI_SERVICE_ACCOUNT }}
           export_default_credentials: true
       - run: |
-          go install github.com/google/go-containerregistry/cmd/crane
+          go install github.com/google/go-containerregistry/cmd/crane@v0.6.0
```

Comment:
Why not pin by SHA go install github.com/google/go-containerregistry/cmd/crane@092caf0f50b9a09588b653e08480ec4077cb289f? I understand it is hard to know the version by SHA. But a comment with a version number would address that issue.

Reply:
Practically speaking, there's no reason to prefer the SHA. Once the Go module proxy has seen a commit tagged as v0.6.0, it will cache and serve that content even if the v0.6.0 tag pointer moves to point to another commit.

So this is effectively the same as a SHA, and provides useful semantic information to readers.